infrastructure/terraform/components/reporting/scripts/sql/ingestion/request_item_status.sql

```diff
@@ -26,7 +26,9 @@ USING (
         status,
         failedreason,
         patientodscode,
-        CAST("$classification".timestamp AS BIGINT) AS timestamp
+        CAST("$classification".timestamp AS BIGINT) AS timestamp,
+        sendinggroupname,
```

Comment:
Can we keep these 2 new columns together with the other sending group columns (as per report SQL and table DDL)